src/main/java/at/ac/tuwien/kr/alpha/grounder/RuleGroundingOrder.java

```diff
@@ -0,0 +1,165 @@
+package at.ac.tuwien.kr.alpha.grounder;
+
+import at.ac.tuwien.kr.alpha.common.atoms.BasicAtom;
+import at.ac.tuwien.kr.alpha.common.atoms.Literal;
+import at.ac.tuwien.kr.alpha.common.terms.VariableTerm;
+
+import java.util.*;
+
+/**
+ * Provides the grounder with information on the order to ground the literals in the body of a rule.
+ * Copyright (c) 2017, the Alpha Team.
+ */
+public class RuleGroundingOrder {
```

Comment:
It would be nice to have a few sentences describing how the order is actually established. Just by looking at the implementation I am having a hard time to reconstruct the big picture.